src/Payments/CardPayment.res

```diff
@@ -88,7 +88,7 @@ let make = (
     None
   }, (empty, complete))
 
-  let (savedMethods, isGuestCustomer) = React.useMemo1(() => {
+  let (_, isGuestCustomer) = React.useMemo1(() => {
```

Comment:
If we are not going to use this savedMethods, please remove it from here

Reply:
Please remove this _ as well. useMemo should only return isGuestCustomer

Reply:
Updated